src/util.h

```diff
@@ -1,30 +1,88 @@
 #include <fstream>
+#include <string.h>
+#include "solver.h"
 
-void file_to_grids(std::string filename, std::vector<Grid> &grids){
+
+struct StartupOptions
+{
+    std::string input_file = "simple-500";
+    Solver *solver = new BFS();
+    bool run_parallel = true;
```

Comment:
I think we should change this to run_parallel_only.
The default should be we run both the sequential and parallel. Down the line, it seems unlikely that we will run the sequential a lot. Though another alternative is just to have more options for this: parallel_only, seq_only, both.